code/app/build.gradle.kts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import java.util.Properties
+
+plugins {
+    id("com.android.application")
+    id("com.google.gms.google-services")
+    id("org.jetbrains.kotlin.android")
+    kotlin("plugin.compose") version "2.0.0" apply false
+    kotlin("plugin.serialization") version "1.6.0" apply false
+}
+
+val mavenCoreVersion: String by project
+val mavenEdgeIdentityVersion: String by project
+
+android {
+    namespace = "com.adobe.marketing.mobile.aepsdkbcapp"
+    compileSdk = 35
+
+    defaultConfig {
+        applicationId = "com.adobe.marketing.mobile.aepsdkbcapp"
+        minSdk = 24
```

Comment:
Let us keep it consistent with the SDK extension min version 21.
Additionally, lets use everything from AEP Gradle Plugin. Example:  https://github.com/adobe/aepsdk-assurance-android/blob/main/code/assurance-testapp/build.gradle.kts